core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java

```diff
@@ -347,9 +347,9 @@ void shouldNormalizeAmpersand(Mode mode) {
     @Test
     void serverShouldRemoveMatrixVariablesWhenNotAllowed() {
         // Not allowed
-        assertAccepted(forServer("/;a=b?c=d;e=f"), "/", "c=d;e=f");
```

Comment:
Actually, why is this failing?

Reply:
Matrix variables are followed by a path variable so it shouldn't be right after the slash. Isn't it right?

Reply:
Sorry, I left a comment on the wrong line 😅    I actually meant L352.

Reply:
It isn't failed. 😅 Let me revert the change. 😉